### DISCUSSION TITLE
Godot Visual Sky Shader

### DISCUSSION THREAD

**Discussion by KC098** [godotengine/godot-proposals#4275] [Editor]
The newly integrated sky shader is pretty neat. I'm very familiar with using nodes, in fact I pretty much use nodes for shading. Writing shaders, while It can be or is the most convenient way of doing shaders. Visual sky shaders should be a thing. 
I imagine is not as complex to make when there's already a visual shader editor, but I can be wrong.
![image](https://user-images.githubusercontent.com/73260374/160062214-f723ea59-1799-463a-a4fc-34f78b6faf79.png)
[picture taken using Godot 4 alpha 5]

**Comment by clayjohn** [comment]
This is already supported. If you click "New Shader" you get the shader creation popup and you can choose to make a visual shader sky shader.
